### ISSUE TITLE
CondFD questions - TOld/TDold and other flavors of T

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5332]
While working on #5297 to add Conduction finite different heat flux outputs, noticed some things.
- [X] The engineering reference says "The grid for each material is established by specifying a half node for each edge of a material and equal size nodes for the rest of the material."  But the eio node position output and the new heat capacity outputs appear to show that all nodes in a given material layer are of equal size (no half-thickess nodes).
  _MJW-23Nov2015-The node positions in the eio are correct, and the half-nodes are implemented as stated in the engineering reference._
- [ ] The use of SurfaceFD( Surf ).TOld and TDold is questionable.  These are only set to TempInitValue and never change, but then they are used to initialize several other variables every timestep in [InitHeatBalanceFiniteDiff](https://github.com/NREL/EnergyPlus/blob/095b778013f3790a893d738d204b56e6faf0a771/src/EnergyPlus/HeatBalFiniteDiffManager.cc#L587-L602)

```
            SurfaceFD( SurfNum ).T = SurfaceFD( SurfNum ).TOld;
            SurfaceFD( SurfNum ).Rhov = SurfaceFD( SurfNum ).RhovOld;
            SurfaceFD( SurfNum ).TD = SurfaceFD( SurfNum ).TDOld;
            SurfaceFD( SurfNum ).TDT = SurfaceFD( SurfNum ).TDreport; //PT changes from TDold to TDreport
            SurfaceFD( SurfNum ).TDTLast = SurfaceFD( SurfNum ).TDOld;
```

At one time TDOld was updated, but that [code](https://github.com/NREL/EnergyPlus/blob/095b778013f3790a893d738d204b56e6faf0a771/src/EnergyPlus/HeatBalFiniteDiffManager.cc#L1200-L1201) has been changed to use TDreport instead.

```
        //  SurfaceFD(Surf)%TDOld          = SurfaceFD(Surf)%TDT
        TDreport = TDT;
```
- [ ] In general, there are so many flavors of T (most of which are not even defined with a comment) that it's difficult to follow what they are and how they are used.  From [HeatBalFiniteDiffManager.hh](https://github.com/NREL/EnergyPlus/blob/095b778013f3790a893d738d204b56e6faf0a771/src/EnergyPlus/HeatBalFiniteDiffManager.hh#L127-L148)

```
    struct SurfaceDataFD
    {
        // Members
        Array1D< Real64 > T;
        Array1D< Real64 > TOld;
        Array1D< Real64 > TT;
        Array1D< Real64 > Rhov;
        Array1D< Real64 > RhovOld;
        Array1D< Real64 > RhoT;
        Array1D< Real64 > TD;
        Array1D< Real64 > TDT;
        Array1D< Real64 > TDTLast;
        Array1D< Real64 > TDOld;
        Array1D< Real64 > TDreport;
        Array1D< Real64 > RH;
        Array1D< Real64 > RHreport;
        Array1D< Real64 > EnthOld; // Current node enthalpy
        Array1D< Real64 > EnthNew; // Node enthalpy at new time
        Array1D< Real64 > EnthLast;
        int GSloopCounter; // count of inner loop iterations
        int GSloopErrorCount; // recurring error counter
        Real64 MaxNodeDelTemp; // largest change in node temps after calc
```

**Comment by mjwitte** [comment]
- [ ] Surface Outside Face Conduction Heat Transfer Rate per Area sees to have the wrong sign for adiabatic surfaces. For example file CondFD1ZonePurchAirAutoSizeWithPCM.idf 

![image](https://cloud.githubusercontent.com/assets/4991075/11355493/fc63c668-921a-11e5-92b6-0d7a517672b8.png)

The sign convention for Outside Face Conduction:
A positive value means that the conduction is from just inside the outside face toward the outside face. A negative value means that the conduction is from the outside face into the core of the heat transfer surface.

The sign convection for Inside Face Conduction:
A positive value means that the conduction is from just inside the inside face toward the inside face. A negative value means that the conduction is from the inside face into the core of the heat transfer surface.

So, it seems if the inside face and outside face of an adiabatic surface are supposed to be symmetrical, then both of these values should have the same sign.  Should check this with some non-CondFD surfaces and see if that holds true.

**Comment by Fan-Feng** [comment]
Yes, I also found this is questionable, and I add two output variables: Phase Change Node Enthalpy and  Phase Change Node Previous Enthalpy. Then I found the initialization of EnthOld is also questionable..
